### ISSUE TITLE
[Bug] Allow user to select multiple times the same pivot

### ISSUE THREAD

**Issue by rezahmady** [Laravel-Backpack/CRUD#5707]
after fix this reported issue [5635](https://github.com/Laravel-Backpack/CRUD/issues/5635)
 this feature stopped working 
[Allow user to select multiple times the same pivot](https://backpackforlaravel.com/docs/6.x/crud-fields#allow-user-to-select-multiple-times-the-same-pivot)

**Comment by pxpm** [comment]
Hello @rezahmady thanks for the report. 

Can you provide some reproduction steps ? (ideally a simple repository we can clone to test your code with only the relevant model/migrations and crud controllers to reproduce this scenario). 

We have some tests in place, and they are passing https://github.com/Laravel-Backpack/CRUD/blob/dee2eeca96190ade092bc2777c95275ec3d4130c/tests/Unit/CrudPanel/CrudPanelCreateTest.php#L795

Cheers

**Comment by rezahmady** [comment]
my relation in Video model:

```
public function crew_people(): \Illuminate\Database\Eloquent\Relations\BelongsToMany
    {
        return $this->belongsToMany(CrewPeople::class,'crew_people_video', 'video_id', 'crew_people_id')
            ->withPivot(['crew_role_id', 'id', 'star']);
    }
```

this is my field 


```

[
                [
                    'name'          => 'crew_people',
                    'type'          => "relationship",
                    'label' => '',
                    'allow_duplicate_pivots' => true,
                    'new_item_label'  => trans('crew::lang.add_crew_people'),
                    'tab' => trans('crew::lang.crew_people_plural'),
                    'subfields'   => [
                        [
                            'name' => 'crew_role_id',
                            'label' => trans('crew::lang.crew_role_singular'),
                            'type' => "select2_from_array",
                            'options' => CrewRole::pluck('name', 'id')->toArray(),
                            'wrapper' => [
                                'class' => 'form-group col-md-4',
                            ],
                            'placeholder' => trans('crew::lang.crew_role_plural_placeholder'),
                        ],
                        [
                            'name' => 'star',
                            'label' => trans('crew::lang.crew_star'),
                            'type' => "switch",
                            'wrapper' => [
                                'class' => 'form-group col-md-2 pt-5',
                            ],
                        ]
                    ],
                    'pivotSelect'=> [
                        // 'attribute' => "title", // attribute on model that is shown to user
                        'label' => trans('crew::lang.crew_people_singular'),
                        'placeholder' => trans('crew::lang.crew_placeholder'),
                        'wrapper' => [
                            'class' => 'col-md-6',
                        ],
                        'inline_create' => true,
                        // note that just like any other select, enabling ajax will require you to provide an url for the field
                        // to fetch available options. You can use the FetchOperation or manually create the enpoint.
                        'ajax' => true,
                        'minimum_input_length'    => 0,
                        'data_source' => url('/api/crew/people'),
                    ],
                ]
            ]

```
  but result 


![image](https://github.com/user-attachments/assets/0fd26a56-6ebb-4332-bd3c-1528f61401c5)

    
  

 I can't choose Eddie (first item) twice


whene i remove this line in field properties is correct

```
'inline_create' => true,
```

the main problem is inline create

**Comment by pxpm** [comment]
Thanks @rezahmady I was able to reproduce and fix the issue. 

I've just released backpack/pro 2.2.28 with the fix. Please allow a few moments for our private repository to pick up the changes and you should be able to get the fixed version 👍 

Let us know if you are still experiencing issues. 

Cheers

**Comment by cod3rshotout** [comment]
@pxpm I've the latest version of backpack and now I'm getting this: jQuery.Deferred exception: $allowDuplicatePivots is not defined ReferenceError: $allowDuplicatePivots is not defined

The structure is the following:

    CRUD::field([
        'name' => 'paths',
        'label' => __('backend/museum.info.paths'),
        'type' => 'relationship',
        'entity' => 'paths',
        'attribute' => 'name',
        'pivot' => true,
        'multiple' => true,
        'model' => 'App\Models\MuseumPath',
        'inline_create' => ['entity' => 'museum-path'],
        'wrapper' => [
            'class' => 'form-group col-md-12'
        ],
        'subfields' => [],
        'pivotSelect' => [
            'label' =>  __('backend/museum.paths.select_path'),
            'inline_create' => [
                'entity' => 'museum-path',
                'modal_route' => route('museum-path-inline-create'),
            ],
            'data_source' => route('museum.fetchMuseumPaths'),
        ],
        'tab' => $paths_tab,
    ]);

    public function paths()
    {
        return $this->belongsToMany(MuseumPath::class, 'museum_museum_path');
    }
